includes/discovery/bgp-peers.inc.php

```diff
@@ -125,12 +125,16 @@
                     }
                 }
 
-                $af_query = "SELECT bgpPeerIdentifier, afi, safi FROM bgpPeers_cbgp WHERE `device_id` = '".$device['device_id']."' AND bgpPeerIdentifier = '".$peer['ip']."'";
-                foreach (dbFetchRows($af_query) as $entry) {
+                $af_query = "SELECT bgpPeerIdentifier, afi, safi FROM bgpPeers_cbgp WHERE `device_id`=? AND bgpPeerIdentifier=? AND context_name=?";
+                foreach (dbFetchRows($af_query, [$device['device_id'], $peer['ip'], $device['context_name']]) as $entry) {
                     $afi  = $entry['afi'];
                     $safi = $entry['safi'];
                     if (!$af_list[$entry['bgpPeerIdentifier']][$afi][$safi]) {
-                        dbDelete('bgpPeers_cbgp', '`device_id` = ? AND `bgpPeerIdentifier` = ? AND afi=? AND safi=?', array($device['device_id'], $peer['ip'], $afi, $safi));
+                        dbDelete(
+                            'bgpPeers_cbgp',
+                            '`device_id`=? AND `bgpPeerIdentifier`=? AND context_name=? AND afi=? AND safi=?',
+                            [$device['device_id'], $peer['ip'], $device['context_name'], $afi, $safi]
```

Comment:
Is that the first time we've had an array shorthand in since the split :D

Reply:
No, there is some in the processor code I think.  Not ready for a mass replace, but I figured I might as well use it in new/changed code.  New is probably safer in case we need to backport.